sudoku/BUILD

```diff
@@ -0,0 +1,81 @@
+#
```

Comment:
The rest of the examples repository uses a two-level nesting.
What should the top-level package be for this one?

Reply:
We decided it should be `games`